ground/gcs/src/plugins/setupwizard/pages/inputpage.ui

```diff
@@ -447,6 +447,48 @@ p, li { white-space: pre-wrap; }
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QToolButton" name="tbscrossfireButton">
+       <property name="font">
+        <font>
+         <pointsize>10</pointsize>
+        </font>
+       </property>
+       <property name="toolTip">
+        <string>Team BlackSheep Crossfire</string>
+       </property>
+       <property name="styleSheet">
+        <string notr="true">QToolButton { border: none }</string>
+       </property>
+       <property name="text">
+        <string>Crossfire</string>
+       </property>
+       <property name="icon">
+        <iconset resource="../wizardResources.qrc">
+         <normaloff>:/setupwizard/resources/bttn-tbscrossfire-up.png</normaloff>
+         <normalon>:/setupwizard/resources/bttn-tbscrossfire-down.png</normalon>
+         <disabledoff>:/setupwizard/resources/bttn-tbscrossfire-disabled.png</disabledoff>:/setupwizard/resources/bttn-tbscrossfire-up.png</iconset>

```

Comment:
Just did a compare, it looks the same as PPM?
Have to admit, did a manual copy/paste.

What is not looking right to you?